.babelrc.js

```diff
@@ -1,6 +1,6 @@
 const config = {
   presets: [
-    '@babel/preset-env',
```

Comment:
What is the impact of this? is this acually a change or is the default different for linux vs windows?

Reply:
The solution was from here I think: https://github.com/webpack/webpack/issues/4039